CMakeLists.txt

```diff
@@ -6,7 +6,11 @@ if(NOT DEFINED CMAKE_MACOSX_RPATH)
   set(CMAKE_MACOSX_RPATH 0)
 endif()
 
-set(CMAKE_CXX_STANDARD 17)
+option(GTSAM_NO_BOOST_CPP17 "Require and use boost" ON)
+
+if (GTSAM_NO_BOOST_CPP17)
```

Comment:
Sorry for the misunderstanding, but we will move to c++17 no matter what, so this ifdef is not needed. Also, I'd like to remove this flag completely - we switch to c++17 using the build flags.

Reply:
It will go away